src/main/java/com/matt/forgehax/util/mod/ToggleMod.java

```diff
@@ -27,6 +28,22 @@ public ToggleMod(Category category, String modName, boolean defaultValue, String
                   }
                 })
             .build();
+    this.nolist =
+        getCommandStub()
+            .builders()
+            .<Boolean>newSettingBuilder()
+            .name("hidden")
+            .description("Hides the mod from modlist")
+            .defaultTo(defaultValue)
+            .changed(
+                cb -> {
```

Comment:
This probably shouldn't be enabling/disabling the mod...

Reply:
Indeed, sorry! Fixed